services/user-svc/internal/repositories/organisation_repository.go

```diff
@@ -20,8 +21,48 @@ func OrganisationRepo(logCtx context.Context) *OrganisationRepository {
 	}
 }
 
-func (r *OrganisationRepository) IsAdminInOrganization(organizationID uuid.UUID, userID uuid.UUID) (bool, error) {
+func (r *OrganisationRepository) IsInOrganization(organizationID uuid.UUID, userID uuid.UUID) (bool, error) {
+	membership := models.Membership{
+		UserID:         userID,
+		OrganizationID: organizationID,
+	}
+
+	err := r.db.First(&membership).Error
+	if errors.Is(err, gorm.ErrRecordNotFound) {
+		return false, nil
+	}
+	if err != nil {
+		if hwgorm.IsOurFault(err) {
+			return false, status.Error(codes.Internal, err.Error())
+		} else {
+			return false, status.Error(codes.InvalidArgument, "not a member of this organization")
```

Comment:
why is this an error? we have an answer

Reply:
I have updated it in the way I intended the error handling